website/docs/r/security_group.html.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "scaleway"
 page_title: "Scaleway: security_group"
-sidebar_current: "docs-scaleway-resource-security_group"
+sidebar_current: "docs-scaleway-resource-security-group-x"
```

Comment:
why the `-x` suffix?

Reply:
otherwise `...-security-group-rule` causes this highlight to be caught; we tend to use `-x` as a suffix to work around this (but any string will do)

Reply:
Thank you for the explanation :)